docs/source/examples.select_fps.ipynb

```diff
@@ -29,52 +29,55 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "import wfl, os, sys, yaml\n",
+    "import wfl, pathlib\n",
+    "import yaml\n",
     "from wfl.configset import ConfigSet, OutputSpec\n",
     "from wfl.descriptors.quippy import calc as calc_descriptors\n",
-    "from wfl.select.by_descriptor import greedy_fps_conf_global as get_fps\n",
+    "from wfl.select.by_descriptor import greedy_fps_conf_global\n",
     "\n",
-    "work_dir = os.path.join(os.path.dirname(wfl.__file__), \"../docs/source/examples_files/select_fps\") # Location of I/O files\n",
+    "work_dir = pathlib.Path(wfl.__file__).parents[1].joinpath(\"docs/source/examples_files/select_fps\")\n",
     "\n",
     "# Step 1: Assign descriptors to the database\n",
-    "md        = ConfigSet(os.path.join(work_dir, \"md.traj\"))\n",
-    "md_desc   = OutputSpec(files=os.path.join(work_dir, \"md_desc.xyz\"))\n",
+    "md        = ConfigSet(work_dir.joinpath(\"md.traj\"))\n",
```

Comment:
Do you want to take advantage of one of the nicer things about `Path` syntax, namely the `/` operator, so `work_dir / "md.traj"`

Reply:
Nice to know. Added now.